### ISSUE TITLE
Publishing docs on release does not work

### ISSUE THREAD

**Issue by johanandren** [akka/akka-persistence-spanner#87]
https://travis-ci.com/github/akka/akka-persistence-spanner/jobs/329816006

```
$ eval "$(ssh-agent -s)" && cp .travis/id_rsa /tmp/id_rsa && chmod 600 /tmp/id_rsa && ssh-keygen -p -P "$DEPLOY_PASSPHRASE" -N "" -f /tmp/id_rsa && ssh-add /tmp/id_rsa && sbt -jvm-opts .jvmopts-travis docs/publishRsync
221Agent pid 4572
222Failed to load key /tmp/id_rsa: incorrect passphrase supplied to decrypt private key
223The command "eval "$(ssh-agent -s)" && cp .travis/id_rsa /tmp/id_rsa && chmod 600 /tmp/id_rsa && ssh-keygen -p -P "$DEPLOY_PASSPHRASE" -N "" -f /tmp/id_rsa && ssh-add /tmp/id_rsa && sbt -jvm-opts .jvmopts-travis docs/publishRsync" exited with 255.
```

**Comment by johanandren** [comment]
I dreamed up that we fixed this but it is still broken. Wrong pwd for the private key in the travis conf I guess.

**Comment by johanandren** [comment]
https://travis-ci.com/github/akka/akka-persistence-spanner/jobs/472049303

**Comment by ennru** [comment]
Fixed with #134, I believe.
